cplusplus.json

```diff
@@ -25,5 +25,119 @@
+  {"id": "cpp.maximilian", "name": "Maximilian", "tech": ["computational-science"], "tags": ["dsp", "audio", "fft"], "repo": "micknoise/Maximilian",
+  "imports": ["maximilian.h", "libs/maxim.h"], "examples": ["https://github.com/micknoise/Maximilian/tree/master/maximilian_examples"]},
+  {"id": "cpp.openal-soft", "name": "OpenAL Soft", "tech": ["audio"], "tags": ["3D audio", "audio-synthesis", "audio-game"], "repo": "kcat/openal-soft",
+  "imports": ["al.h", "AL/alc.h", "AL/alext.h", "AL/efx-presets.h"], "examples": ["https://github.com/kcat/openal-soft/tree/master/examples"]},
+  {"id": "cpp.opus", "name": "Opus", "tech": ["audio"], "tags": ["codec"], "repo": "xiph/opus",
+  "imports": ["opus.h"], "examples": ["https://github.com/xiph/opus/tree/master/tests"]},
+  {"id": "cpp.sela", "name": "SELA", "tech": ["audio"], "tags": ["codec"], "repo": "sahaRatul/sela",
+  "imports": ["lpc.h", "rice.h", "apev2.h"], "examples": ["https://github.com/sahaRatul/sela/tree/latest/tests"]},
+  {"id": "cpp.soloud", "name": "SoLoud", "tech": ["audio"], "tags": ["audio-game", "3D audio"], "repo": "jarikomppa/soloud",
+  "imports": ["soloud.h", "soloud_sfxr.h", "soloud_speech.h"], "examples": ["https://github.com/jarikomppa/soloud/tree/master/demos"]},
+  {"id": "cpp.speex", "name": "Speex", "tech": ["audio"], "tags": ["codec"], "repo": "xiph/Speex",
+  "imports": ["speex.h"], "examples": []},
+  {"id": "cpp.tonic", "name": "Tonic", "tech": ["audio"], "tags": ["audio-synthesis"], "repo": "TonicAudio/Tonic",
+  "imports": ["Tonic.h", "RtAudio.h"],"examples": ["https://github.com/TonicAudio/Tonic/tree/master/examples"]},
+  {"id": "cpp.vorbis", "name": "Vorbis", "tech": ["audio"], "tags": ["codec"], "repo": "xiph/vorbis",
+  "imports": ["vorbis/vorbisenc.h", "vorbis/codec.h"], "examples": ["https://github.com/xiph/vorbis/tree/master/examples"]},
+  {"id": "cpp.minimp3", "name": "minimp3", "tech": ["audio"], "tags": ["codec"], "repo": "lieff/minimp3",
+  "imports": ["minimp3.h"], "examples": []},
+
+  {"id": "cpp.libsequence", "name": "libsequence", "tech": ["computational-science"], "tags": ["genetics", "analysis", "high-performance"], "repo": "molpopgen/libsequence", 
+  "imports": ["Sequence/Fasta.hpp", "Sequence/SeqUtilities.hpp", "Sequence/Correlations.hpp"], "examples": ["https://github.com/molpopgen/libsequence/tree/master/examples"]},
+  {"id": "cpp.seqan", "name": "SeqAn", "tech": ["data-structures", "algorithms"], "tags": ["high-performance"], "repo": "seqan/seqan",
+  "imports": ["seqan/align.h", "seqan/stream.h", "seqan/sequence.h"], "examples": ["https://github.com/seqan/seqan/tree/master/demos"]},
+  {"id": "cpp.vcflib", "name": "vcflib", "tech": ["parsing"], "tags": ["vcf"], "repo": "vcflib/vcflib",
+  "imports": ["Variant.h"], "examples": ["https://github.com/vcflib/vcflib/tree/master/test/tests"]},
+  {"id": "cpp.wham", "name": "wham", "tech": ["computational-science"], "tags": ["genetics", "analysis",], "repo": "zeeev/wham",
```

Comment:
i don't think it's a library. this seems to be a command line tool. we usually exclude those.